aws-efs-volume/outputs.tf

```diff
@@ -0,0 +1,12 @@
+// HACK(el): we do this to hint TF dependency graph since modules can't depend_on
+output "id" {
+  value = aws_efs_file_system.efs.id
+}
+
+output "dns_name" {
```

Comment:
Will there be different dns names per subnet?

Reply:
According to some blog posts, the magic VPC DNS resolution should make the same name work (and resolve to different IP's) in all subnets, but I'm going to validate that. Seems fishy to me.

Reply:
Learned more when testing this module further: Even though the mount point in each subnet has its own IP, there's only one DNS name for the volume. So we only need to output one DNS name from the module.

The individual mount point IP's *could* be useful outputs if someone's getting tricky with networking, but imnsho we can keep the interface simple for now and expand it if we need more detail.